tensorflow/lite/tools/versioning/op_version.cc

```diff
@@ -513,6 +513,11 @@ int GetBuiltinOperatorVersion(const OpSignature& op_sig) {
       return 1;
 
     case BuiltinOperator_GATHER_ND:
+      if (!op_sig.input_types.empty() &&
+          (op_sig.input_types.at(0) == TensorType_INT8 ||
```

Comment:
I think Integer 8 is already supported in the previous version.

Reply:
INT8 was part of the kernel, but it wasn't enabled (attempting to quantize it would give a "quantization not yet supported for GATHER_ND op" error). Support is added in this version by registering the GATHER_ND operator.

Reply:
That is right. Conceputally, this op version table describes the op kernel implementation not the quantizer and converter's behavior. So I would suggest removing the int8 check here since the version two op kernel is able to handle int8 inputs anyway.

Reply:
@abattery Thanks for the clarification. I removed the check and also updated op_version_test accordingly (as well as removing a duplicate test I had added).